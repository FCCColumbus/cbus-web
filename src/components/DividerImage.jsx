function DividerImage() {
  return <img src='/divider.jpg' alt='code on a screen. mostly css' />
}

export default DividerImage;