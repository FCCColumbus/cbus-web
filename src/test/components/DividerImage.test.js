import rendered from "react-test-renderer";

import DividerImage from "../../components/DividerImage";

describe("DividerImage", () => {
  it("renders", () => {
    const view = rendered.create(<DividerImage />);
    expect(view).toMatchSnapshot();
  });
});
