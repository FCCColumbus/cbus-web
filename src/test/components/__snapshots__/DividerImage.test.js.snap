// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DividerImage renders 1`] = `
<img
  alt="code on a screen. mostly css"
  src="/divider.jpg"
/>
`;
